builder/role_image.go

```diff
@@ -146,6 +223,7 @@ func (r *RoleImageBuilder) CreateDockerfileDir(role *model.Role, jsonSpecsDir st
 	if err := os.MkdirAll(jobsDir, 0755); err != nil {
 		return "", err
 	}
+	jsonSpecsDir := filepath.Join(rootDir, "opt/hcf/specs")
```

Comment:
Err, sorry, you're right; per job.

I can move it out if desired, but we generate them all in one go anyways, so doing it per-role-image seemed pointless.